index.js

```diff
@@ -0,0 +1,132 @@
+const inquirer = require("inquirer");
+const Manager = require("./src/Manager");
+const Engineer = require("./src/Engineer");
+const Intern = require("./src/Intern");
+const generateHTML = require("./src/generateHTML");
+const fs = require("fs");
+
+const team = [];
+
+const createAnotherEmployeeQuestion = {
+  type: "list",
+  message: "Select option:",
+  name: "option",
+  choices: ["Engineer", "Intern", "Done"],
+};
+
+const managerQuestions = [
+  {
+    type: "input",
+    message: "Name of the manager?",
+    name: "name",
+  },
+  {
+    type: "input",
+    message: "Employees ID?",
+    name: "id",
+  },
+  {
+    type: "input",
+    message: "Email addres?",
+    name: "email",
+  },
+  {
+    type: "input",
+    message: "Office number?",
+    name: "officeNumber",
+  },
+  createAnotherEmployeeQuestion,
+];
+
+const engineerQuestions = [
+  {
+    type: "input",
+    message: "Name of the engineer?",
+    name: "name",
+  },
+
+  {
+    type: "input",
+    message: "Engineer ID?",
+    name: "id",
+  },
+  {
+    type: "input",
+    message: "Email addres?",
+    name: "email",
+  },
+  {
+    type: "input",
+    message: "GitHub username?",
+    name: "gitHub",
+  },
+  createAnotherEmployeeQuestion,
+];
+
+const internQuestions = [
+  {
+    type: "input",
+    message: "Name of the intern?",
+    name: "name",
+  },
+
+  {
+    type: "input",
+    message: "Intern ID?",
+    name: "id",
+  },
+  {
+    type: "input",
+    message: "Email addres?",
+    name: "email",
+  },
+  {
+    type: "input",
+    message: "Visited school?",
+    name: "school",
+  },
+  createAnotherEmployeeQuestion,
+];
+
+function writeToFile(fileName, answers) {
+  fs.writeFile(fileName, answers, (error) => {
+    error && console.error("error", error);
+  });
+}
+
+async function start() {
+  let answers = await inquirer.prompt(managerQuestions);
```

Comment:
these varaibles are only created once - when the function runs. When the function is done, they are destroyed and recreated from fresh if it is called again. 
This means that you can use const in place of let here as during the lifespan of the function you'll never need to reassign the values. 
Hope this makes sense, but drop me a message if not and we can have a quick chat if you like :-)